src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/driver_add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String driverName = req.getParameter("name");
+        String licenseNumber = req.getParameter("licenseNumber");
+        Driver driver = new Driver();
+        driver.setName(driverName);
+        driver.setLicenseNumber(licenseNumber);
+        driverService.create(driver);
+        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
```

Comment:
it's not very convenient to user to go to index page after some add operation, let's redirect him to page where he can see result of his operation